### ISSUE TITLE
Build your own GPT-3 AI app with Next.js and OpenAI

### ISSUE THREAD

**Issue by saliksik** [Nutlope/passive-to-active-voice#1]
Hi Sir!

Just watched your new YouTube video (Build your own GPT-3 AI app with Next.js and OpenAI).

Hoping you can also update the codebase of this web app just like with your twitter bio that streams the response from OpenAI.

Thank you.

**Comment by Nutlope** [comment]
Good point, I should probably do that. Feel free to submit a PR for this btw! Otherwise, I can probably tackle it later.

**Comment by saliksik** [comment]
I would love to write a pull request, but I am still learning more about React and NextJS. 

My only suggestion for this web app is, if possible, to not include a dropdown option or slice function, so that I can compare the two codebases (twitterbio and this) and learn more about how they work.

Thank you, Sir Hassan!
